arctic/arctic.py

```diff
@@ -183,13 +188,17 @@ def __getstate__(self):
     def __setstate__(self, state):
         return Arctic.__init__(self, **state)
 
-    @mongo_retry
-    def list_libraries(self):
+    def list_libraries(self, newer_than_secs=120):
```

Comment:
It seems like there are a lot of dials for caching.  I think it's worth defaulting to the happy cache path.

e.g. This list_libraries seems useful to have a way to invalidate, but could be defaulted to -1.  That the cache value is always assumed to be ok.

It seems like there are a lot of ways e.g. mismatch between values where your cache will get ignored.

Reply:
Makes sense. Will default to the cached version with the default in config also to enable cache